libecs/include/pivot/ecs/Core/Data/asset.hxx

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <string>
+
+namespace pivot::ecs::data
+{
+struct Asset {
+    std::string name;
+
```

Comment:
Maybe add an enum with the "type" of the asset (material, assets, etc...) ?

Reply:
Not necessary for now, and would tie in too much graphical things in the ECS.